package.json

```diff
@@ -125,6 +125,10 @@
           "value": "zh-tw",
           "description": "繁體中文 (zh-tw)"
         },
+        {
```

Comment:
Please insert `uk` section in alphabetic order;
in this case after `th` section.